### DISCUSSION TITLE
this is throwing error while setup. I am using NextJS 15.0.3 and react 19

### DISCUSSION THREAD

**Discussion by tmlkmn** [stack-auth/stack-auth#341] [Show and tell]
ready to install Stack? Yes

Installing dependencies...

npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: kanban@0.1.0
npm error Found: react@19.0.0-rc-66855b96-20241106
npm error node_modules/react
npm error   react@"19.0.0-rc-66855b96-20241106" from the root project
npm error   peer react@"^18.2.0 || 19.0.0-rc-66855b96-20241106" from next@15.0.3
npm error   node_modules/next
npm error     next@"15.0.3" from the root project
npm error     peer next@">=14.1" from @stackframe/stack@2.6.21
npm error     node_modules/@stackframe/stack
npm error       @stackframe/stack@"*" from the root project
npm error   1 more (react-dom)
npm error
npm error Could not resolve dependency:
npm error peer react@">=18.2" from @stackframe/stack@2.6.21
npm error node_modules/@stackframe/stack
npm error   @stackframe/stack@"*" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error
npm error
npm error For a full report see:
npm error C:\Users\temel\AppData\Local\npm-cache\_logs\2024-11-14T09_41_27_789Z-eresolve-report.txt
npm error A complete log of this run can be found in: C:\Users\temel\AppData\Local\npm-cache\_logs\2024-11-14T09_41_27_789Z-debug-0.log
√ Command npm install @stackframe/stack succeeded
Error: Command npm install @stackframe/stack failed with code 1
    at ChildProcess.<anonymous> (file:///C:/Users/temel/AppData/Local/npm-cache/_npx/0a541adb51315b9f/node_modules/@stackframe/init-stack/index.mjs:506:20)
    at ChildProcess.emit (node:events:514:28)
    at ChildProcess._handle.onexit (node:internal/child_process:294:12)

===============================================

An error occurred during the initialization process.

===============================================

If you need assistance, please try installing Stack manually as described in https://docs.stack-auth.com/getting-started/setup or join our Discord where we're happy to help: https://discord.stack-auth.com

Error message: Command npm install @stackframe/stack failed with code 1

**Comment by N2D4** [comment]
Thanks for the report, should be fixed now!

**Reply by tmlkmn** [reply]
İt's working now, thank you so much for your support.
